buildSrc/src/main/java/Repositories.kt

```diff
@@ -30,10 +30,18 @@ fun ScriptHandler.addDefaultInputRepository() {
 
 internal fun RepositoryHandler.addDefaultInputRepository() {
   google()
-  jcenter()
   maven {
     setUrl("https://jitpack.io")
   }
+  jcenter {
+    content {
+      includeGroup("com.mopub")
+      includeGroup("com.criteo.mediation.mopub")
+      includeGroup("com.criteo.mediation.google")
+      includeModule("org.jetbrains.trove4j", "trove4j")
```

Comment:
Could you add a link to the Jetbrains YouTrack ? So we can easily check if migration was done.

Also the CI seems red because of detekt relying on kotlinx.html: https://github.com/Kotlin/kotlinx.html/issues/173

Reply:
done